bower.json

```diff
@@ -27,18 +27,13 @@
   ],
   "dependencies": {
     "polymer": "^1.6.0",
-    "px-theme": "^3.0.0",
-    "px-buttons-design": "^2.0.0",
     "px-deck-selector": "^2.0.0",
     "px-headings-design": "^1.0.0"
```

Comment:
Or are you pointing out that it should be a dev-dependency rather than a dependency?